Plugins/MeshFeatureTracking/src/MeshFeatureTrackingComponent.cpp

```diff
@@ -218,7 +218,7 @@ void MeshFeatureTrackingComponent::setData( const Ra::Engine::Renderer::PickingR
     // check picking mode / render config OK
     if ( data.m_mode == PickingMode::RO || data.m_mode > PickingMode::TRIANGLE ) { return; }
     m_data.m_mode = data.m_mode;
-    m_data.m_data = {-1, -1, -1, -1};
+    m_data.m_data = {uint( -1 ), uint( -1 ), uint( -1 ), uint( -1 )};
```

Comment:
-1 is definitively not an uint. Such a cast is not safe. Prefer `numeric_limits< unsigned int >::max()` if this is what you expect.